src/plccng/Scanner/matcher/matcher_test.py

```diff
@@ -0,0 +1,33 @@
+from ...load_spec.parse_spec import parse_lexical_spec
+from ...load_spec.structs import Line
```

Comment:
This probably makes sense. Maybe we should hoist it another level higher so that it doesn't belong to load_spec or to scanner.